local_planner/include/local_planner/planner_functions.h

```diff
@@ -147,12 +147,14 @@ Eigen::ArrayXf getConicKernel(int radius);
 void printHistogram(const Histogram& histogram);
 
 /**
-* @brief      finds the minimum cost direction in the tree
-* @param[out] p_pol, polar coordinates of the cheapest direction
-* @param[in]  path_node_positions, array of expanded tree nodes
-* @param[in]  position, current vehicle position
+* @brief      Returns a setpoint that lies on the given path
+* @param[in]  vector of nodes defining the path
```

Comment:
Probably mention that the last node should have index 0

Reply:
done